06-wt-multilevel-models-1.Rmd

```diff
@@ -466,13 +466,13 @@ with `sjPlot::tab_model()`, as well as a file name that ends in `.doc` to the
 `filename` argument, i.e.:
 
 ```{r}
-apaTables::apa.reg.table(m_linear)
+(m_summary = apaTables::apa.reg.table(m_linear))
```

Comment:
Thanks. Wondering if we should actually delete this entire chunk (as it's the next chunk which specifies a filename, as the above code suggests)? Or, do you think it's worthwhile to keep this, to show how this output can be saved to an object?